### DISCUSSION TITLE
Avoid deauthenticating users when updating open-webui

### DISCUSSION THREAD

**Discussion by Fusseldieb** [open-webui/open-webui#5545] [Q&A]
As the title suggests, I'm not sure how I can update the container without deauthenticating all users. Everytime I do:
`docker stop xxxxxx && docker rm xxxxx`
and then
`docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -e TZ=America/Sao_Paulo -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

The new instance starts up, but all users get kicked out with a 401 error. They can obviously sign back in, but I would like to know how I can avoid this. Is there any additional volume I need to mount that holds the sessions?

Thanks in advance!

**Comment by tjbck** [comment]
You'd want to specify `WEBUI_SECRET_KEY` using env var here.
